src/moscot/backends/ott/solver.py

```diff
@@ -435,7 +435,7 @@ def _prepare(
         cost_matrix_rank: Optional[int] = None,
         time_scales_heat_kernel: Optional[TimeScalesHeatKernel] = None,
         # problem
-        alpha: float = 0.5,
+        alpha: Optional[float] = None,
```

Comment:
can we just make a comment behind this that default is 0.5

Reply:
I'm actually wondering whether we should have a default `alpha` whenever we don't want it to be 1.0

I.e. always set it explicitly in the classes which use GW, wdyt?

Reply:
I set it like this so that it doesn't change the current behaviour. 
>I.e. always set it explicitly in the classes which use GW, wdyt?

You mean to make non-optional? I personally prefer non-optional parameters, especially if the class is an internal solver. I also think we should rename `GWSolver` to `FGWSolver` (because it technically can solve fgw and gw) and just set `alpha=1` when in a `GWProblem`.

Reply:
I think that makes sense